bunsen-avro-records-stu3/pom.xml

```diff
@@ -144,12 +144,16 @@
             <argument>http://hl7.org/fhir/StructureDefinition/AllergyIntolerance</argument>
             <argument>http://hl7.org/fhir/StructureDefinition/Procedure</argument>
 
+            <argument>http://hl7.org/fhir/StructureDefinition/Medication</argument>
+            <argument>http://hl7.org/fhir/StructureDefinition/MedicationDispense</argument>
             <argument>http://hl7.org/fhir/StructureDefinition/MedicationRequest</argument>
             <argument>http://hl7.org/fhir/StructureDefinition/MedicationStatement</argument>
```

Comment:
Could you also include few more resources down below ?

`<argument>http://hl7.org/fhir/StructureDefinition/ProcedureRequest</argument>
  <argument>http://hl7.org/fhir/StructureDefinition/Encounter</argument>
  <argument>http://hl7.org/fhir/StructureDefinition/CarePlan</argument>
  <argument>http://hl7.org/fhir/StructureDefinition/Claim</argument>
  <argument>http://hl7.org/fhir/StructureDefinition/Immunization</argument>
  <argument>http://hl7.org/fhir/StructureDefinition/EpisodeOfCare</argument>
  <argument>http://hl7.org/fhir/StructureDefinition/Measure</argument>
  <argument>http://hl7.org/fhir/StructureDefinition/MeasureReport</argument>`

Reply:
Added these resources, 44b18b5838cf49bff094330b7ebbe4092c67362f